### ISSUE TITLE
Would like to tackle additional theme options

### ISSUE THREAD

**Issue by bryanbierce** [jeremyramin/terminal-plus#191]
In a follow up to finding an answer to #185 I would love to create and submit a PR for a collection of additional theme options. I would also really like to create the ability to select the ansi colors as a separate theme as well. Are either of these contributions you would be open to accepting?

**Comment by nixpulvis** [comment]
Some packages automatically adjust based on the active theme, it'd be awesome if this package did too.

**Comment by bryanbierce** [comment]
Yeah I'd be happy to try if the maintainer was open to the PR. It looks like they have abandoned the project though. PR's sitting unanswered since December.

**Comment by nixpulvis** [comment]
Make a fork, nothing holding me to this repo directly.

**Comment by The-Penultimate-Defenestrator** [comment]
@bryanbierce It looks like the owner has completely dropped off the face of the earth. [Changes stopped abruptly at the end of last year](https://github.com/jeremyramin/terminal-plus/graphs/contributors) and the author [hasn't contributed once this year](https://github.com/jeremyramin). Maybe we do need a community fork. Someone who understands how this package works enough to make changes should do it.

**Comment by bryanbierce** [comment]
@The-Penultimate-Defenestrator Agreed. I have held off on taking it up because I haven't actually dug into what makes it tick. I poked around enough to add themes but that was it. I am game to dive in and figure it out but I'm starting with a new company this week so it will take me a little while to get the time. You wanna help?

**Comment by The-Penultimate-Defenestrator** [comment]
Sure. I know JavaScript, but not CoffeeScript. I can probably learn though.
